CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Changelog
 
+## [1.62](https://github.com/brave/brave-ios/releases/tag/v1.62)
+
+### Web3
+
+- Implemented Safe Sign for CoW Swaps. [(#8114)](https://github.com/brave/brave-ios/issues/8114)
+- Added support for fetching unknown tokens from non-mainnet transactions. [(#8529)](https://github.com/brave/brave-ios/issues/8529)
+- Added “Creating Wallet” step to Wallet onboarding. [(#8345)](https://github.com/brave/brave-ios/issues/8345)
+- Added “Accounts” and “Transactions” tab to the asset detail view. [(#8137)](https://github.com/brave/brave-ios/issues/8137)
+- Updated Line chart color to match the new Brave Wallet design on Portfolio and Asset details screen. [(#8136)](https://github.com/brave/brave-ios/issues/8136)
+- Updated account blockies to be square similar to desktop. [(#8375)](https://github.com/brave/brave-ios/issues/8375)
+- Migrated Solana Swaps to use Jupiter Swap API v6. [(#8603)](https://github.com/brave/brave-ios/issues/8603)
+- Fixed the approve screen not showing the confirmation when approve limit is lower than transaction value. [(#8721)](https://github.com/brave/brave-ios/issues/8721)
+- Fixed portfolio filter from opening intermittently when scrolling up and down the portfolio asset list. [(#8709)](https://github.com/brave/brave-ios/issues/8709)
+- Fixed UI issues in Brave Wallet resulting from NFT GIFs failing to load. [(#8560)](https://github.com/brave/brave-ios/issues/8560)
+- Fixed Wallet onboarding flow when initiated on two windows in split view on iPad. [(#8425)](https://github.com/brave/brave-ios/issues/8425)
+
+### General
+
+- Added Global Privacy Control toggle under “Brave Shields & Privacy.” [(#8556)](https://github.com/brave/brave-ios/issues/8556)
+- Added “View all version info” to the version context menu under “About” in settings. [(#8705)](https://github.com/brave/brave-ios/issues/8705)
+- Added auto-upgrading for mixed content that contains media using a content blocker. [(#8664)](https://github.com/brave/brave-ios/issues/8664)
+- Added authentication for the user when disabling “Browser Lock” from settings. [(#8608)](https://github.com/brave/brave-ios/issues/8608)
+- Added option in “Search” settings to disable bookmarks and history suggestions. [(#2336)](https://github.com/brave/brave-ios/issues/2336)
+- Updated SSL Certificate display logic to be more in line with Chromium.  [(#8703)](https://github.com/brave/brave-ios/issues/8703)
+- Updated Tracking Protection Stats script to prevent DOS. [(#8562)](https://github.com/brave/brave-ios/issues/8562)
```

Comment:
If this is referring to Denial of Service, I believe it's written as `DoS`. We should probably change the above to the following as some users might not understand what `DOS` means. Example:

```
- Updated Tracking Protection Stats script to prevent Denial of Service (DoS). [(#8562)](https://github.com/brave/brave-ios/issues/8562)
```